### ISSUE TITLE
GeekIt box shows up even on the home page of geeksforgeeks

### ISSUE THREAD

**Issue by deepika087** [ManrajGrover/geekiT#2]
GeekBox shows up in homepage for geeksforgeeks and pages like "http://www.geeksforgeeks.org/page/2/", which is not required because either the checkbox should override the individual questions on that page or it is of no use.

**Comment by manrajgrover** [comment]
@deepika087 

Sorry for late reply. I think we can handle this in [manifest](https://github.com/ManrajGrover/geekiT/blob/master/manifest.json) itself. Did you try changing `matches`?

**Comment by deepika087** [comment]
@ManrajGrover 
Yeah the same idea occurred to me too but this is what I thought.
I used exclude_matches but that didn't work for home page. That is geeksit box was still getting shown on the homepage. Therefore I made the changes as submitted in the Pull request 
![homepageafterexcludematches](https://cloud.githubusercontent.com/assets/6492488/6797123/fd46d3a2-d223-11e4-91ff-601d9cdbb2da.png)
![manifest_exclude_matches](https://cloud.githubusercontent.com/assets/6492488/6797122/fc246cc8-d223-11e4-8b7c-6f1b2d39eeef.png)
